### PR TITLE
Fix UI

### PR DESCRIPTION
**Pull request by shrimeichock**
- put "+" button on checkout page
- Add an icon to browser tab <- only showing up in chrome
- Fix decimal formatting to show 2 decimals when last value is 0
- Fix up formatting of filter options, if possible -> put it beside where books are listed
- Add a small cart symbol for 'total in cart'

image

image

image

image

### CODE REVIEW

File: src/main/resources/templates/checkout.html [hunk 1]
```diff
@@ -46,47 +32,47 @@
...
             <form th:action="@{/checkout}" th:object="${user}" method="post">
                 <div class="input-div">
                     <label for="fullName">Full Name:</label><br>
-                    <input type="fullName" id="fullName" name="fullName" required>
+                    <input type="text" id="fullName" name="fullName" required>
                 </div>
                 <div class="input-div">
                     <label for="emailAddress">Email Address:</label><br>
-                    <input type="emailAddress" id="emailAddress" name="emailAddress" required>
+                    <input type="text" id="emailAddress" name="emailAddress" required>
```
Comment (thanujasiva) [inline src/main/resources/templates/checkout.html]: Why are these all changed to text? Isnt it better if it's email, phone number, etc?

File: src/main/resources/templates/checkout.html [hunk 2]
```diff
@@ -107,12 +93,18 @@ <h2>Cart Information:</h2>
...
-                                <form th:action="@{/removeFromCart}" method="post" th:id="removeFromCartForm">
-                                    <input type="hidden" th:name="'selectedItems'" th:value="${item.getId()}">
-                                    <button type="submit">-</button>
-                                </form>
+                                <div class="add-remove-buttons">
+                                    <form th:action="@{/removeFromCart}" method="post" th:id="removeFromCartForm">
+                                        <input type="hidden" th:name="'selectedItems'" th:value="${item.getId()}">
+                                        <button type="submit">-</button>
+                                    </form>
+                                    <form th:action="@{/addToCart}" method="post" th:id="addToCartForm">
```
Comment (thanujasiva) [inline src/main/resources/templates/checkout.html:99]: Can the addToCart button be moved before the removeFromCart button?

**Comment by sabahsamwatin** [comment]
Checkout page - + buttons' heading says Remove which can be changed as we now have + too

File: src/main/java/bookstore/inventory/Book.java [hunk 3]
```diff
@@ -215,6 +215,10 @@ public Double getPrice() {
         return price;
     }
 
+    public String getFormattedPrice() {
```
Comment (thanujasiva) [inline src/main/java/bookstore/inventory/Book.java:218]: We need to add this to the UML

**Review by thanujasiva — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/checkout.html (hunk 1 above)
Comment (shrimeichock) [reply src/main/resources/templates/checkout.html]: changed email but the rest are not valid types

File: src/main/java/bookstore/inventory/Book.java (hunk 3 above)
Comment (thanujasiva) [reply src/main/java/bookstore/inventory/Book.java:218]: updated the uml

**Review by thanujasiva — APPROVED** [review]

**Pull request merged**